go1.16/darwin/Dockerfile.tmpl

```diff
@@ -8,14 +8,20 @@ RUN \
     && apt-get install -qq -y --no-install-recommends --allow-unauthenticated \
         clang \
         llvm \
+        cmake \
+        patch \
+        python \
+        libssl-dev \
+        libxml2-dev \
+        libz-dev \
     && rm -rf /var/lib/apt/lists/*
 
-ARG OSXCROSS_SDK_URL=https://s3.amazonaws.com/beats-files/deps/MacOSX10.11.sdk.tar.xz
+ARG OSXCROSS_SDK_URL=https://s3.amazonaws.com/beats-files/deps/MacOSX10.13.sdk.tar.xz
 ARG OSXCROSS_PATH=/usr/osxcross
-ARG OSXCROSS_REV=3034f7149716d815bc473d0a7b35d17e4cf175aa
-ARG SDK_VERSION=10.11
-ARG DARWIN_VERSION=15
-ARG OSX_VERSION_MIN=10.10
+ARG OSXCROSS_REV=8a716a43a72dab1db9630d7824ee0af3730cb8f9
+ARG SDK_VERSION=10.13
+ARG DARWIN_VERSION=17
+ARG OSX_VERSION_MIN=10.12
```

Comment:
```
{{if lt .DEBIAN_VERSION "10"}}
ARG OSXCROSS_SDK_URL=https://s3.amazonaws.com/beats-files/deps/MacOSX10.11.sdk.tar.xz
ARG OSXCROSS_PATH=/usr/osxcross
ARG OSXCROSS_REV=3034f7149716d815bc473d0a7b35d17e4cf175aa
ARG SDK_VERSION=10.11
ARG DARWIN_VERSION=15
ARG OSX_VERSION_MIN=10.10
{{ else }}
ARG OSXCROSS_SDK_URL=https://s3.amazonaws.com/beats-files/deps/MacOSX10.13.sdk.tar.xz
ARG OSXCROSS_PATH=/usr/osxcross
ARG OSXCROSS_REV=8a716a43a72dab1db9630d7824ee0af3730cb8f9
ARG SDK_VERSION=10.13
ARG DARWIN_VERSION=17
ARG OSX_VERSION_MIN=10.12
{{ end }}
```

Reply:
this resolve the issue we will have the SDK 10.11 on Debian 7/8/9 and the SDK 10.13 on Debian 10